Makefile

```diff
@@ -14,7 +14,7 @@ OBJECTS = .venv .coverage
 all: install
 install: $(INSTALL_STAMP)
 $(INSTALL_STAMP): $(PYTHON) setup.py
-	$(VENV)/bin/pip install -U pip
+	$(VENV)/bin/pip install -U pip kinto-wizard
```

Comment:
Should we add it to dev-requirements too?

Reply:
We don't have a dev-requirements.txt at the moment, reason why I added it here.